setup.py

```diff
@@ -1,14 +1,11 @@
-import os
 from setuptools import setup, find_packages
+import subprocess
 
 
 def get_version():
-    basedir = os.path.dirname(__file__)
-    with open(os.path.join(basedir, 'rq_dashboard/version.py')) as f:
-        locals = {}
-        exec(f.read(), locals)
-        return locals['VERSION']
-    raise RuntimeError('No version info found.')
+    """Return the latest git tag from repository in cwd."""
```

Comment:
Unless I'm wrong, `setup.py` runs during install. Since the packaged files are **not** a git repo and the users OS might not even have Git available, running `git` during install will result in errors. I think hgtools actually replaces the version getter during build.